vatomcli-plugin-template/src/index.js

```diff
@@ -1,3 +1,8 @@
+import { BasePlugin } from 'vatom-spaces-plugins'
+import { BaseComponent } from 'vatom-spaces-plugins'
+
+BasePlugin
```

Comment:
Is this supposed to be here?

Reply:
OMG! that is so stupid of me. No that shouldn't be there. Sorry.